### ISSUE TITLE
BUG: docs for saving offline charts are wrong

### ISSUE THREAD

**Issue by NickCrews** [moj-analytical-services/splink#1441]
### What happens?

docs at https://moj-analytical-services.github.io/splink/linkerqa.html#splink.linker.Linker.m_u_parameters_chart

say
```
from splink.charts import save_offline_chart
c = linker.match_weights_chart()
save_offline_chart(c.spec, "test_chart.html")
```

but this gives an error `AttributeError: 'HConcatChart' object has no attribute 'spec'`

However, the following works fine to go directly to Altair.Chart.save()? IDK if this is missing something.

```python
c = linker.match_weights_chart()
c.save("match_weights.html")
```

### To Reproduce

see above

### OS:

NA

### Splink version:

main

### Have you tried this on the latest `master` branch?

- [X] I agree

### Have you tried the steps to reproduce? Do they include all relevant data and configuration? Does the issue you report still appear there?

- [X] I agree

**Comment by RossKen** [comment]
I have added a fix for this at #1450 if you want to double-check that it solves your issue @NickCrews?

**Comment by NickCrews** [comment]
Looks good, thanks!
